src/style.css

```
@@ -1,3 +1,78 @@
+html {
+  box-sizing: border-box;
+}
+
 body {
+  --color: #00acdc;
+
+  font-size: 12px;
+  box-sizing: border-box;
+  font-family: Roboto, sans-serif;
+  background: white;
+  margin: 0;
+}
+
+header {
```

Use only class selector
```suggestion
.header {
```

---

Still not fixed